activities/TamTamMicro.activity/app.js

```diff
@@ -47,33 +47,49 @@ enyo.kind({
 		enyo.forEach(this.$.collections.getControls(), function(item) { items.push(item); });
 		for (var i = 0 ; i < items.length ; i++) { items[i].destroy();	}
 
-		// Display collections
-		document.getElementById("body").style.backgroundColor = this.userColor.fill;
-		this.$.collections.applyStyle("background-color", this.userColor.fill);
-		var len = TamTam.collections.length;
-		for(var i = 0 ; i < len ; i++ ) {
-			this.$.collections.createComponent(
-				{ kind: "TamTam.Collection", name: TamTam.collections[i].name, selection: (i == this.collection), ontap: "changeCollection" },
-				{ owner: this }
-			).render();
-		}
-
 		// Remove items
 		var items = [];
-		this.$.items.applyStyle("background-color", this.userColor.fill);
 		enyo.forEach(this.$.items.getControls(), function(item) { items.push(item); });
 		for (var i = 0 ; i < items.length ; i++) { items[i].destroy();	}
+		if(this.pianoMode) {
+			document.getElementById("body").style.backgroundColor = '#ffffff';
+			this.$.collections.applyStyle("background-color", '#ffffff');
 
-		// Display items
-		var collection = TamTam.collections[this.collection];
-		var len = collection.content.length;
-		for(var i = 0 ; i < len ; i++ ) {
-			var item = this.$.items.createComponent(
-				{ kind: "TamTam.Item", name: collection.content[i] },
+			if(currentPianoMode !== null) {
+				this.$.items.createComponent(
```

Comment:
Please use this.userColor.stroke for background of this item.
See [here](https://github.com/llaske/sugarizer/pull/544/files#diff-c2fb8208740e638314fe55c3feeed5b6R90) to see how.